packages/zero-protocol/src/ast.ts

```diff
@@ -320,7 +321,7 @@ type StaticParameter = {
   // Read and write authorization will both send the
   // current authentication data (authData).
   readonly anchor: 'authData' | 'preMutationRow';
-  readonly field: string;
+  readonly field: string | string[];
```

Comment:
Does AST changes require a version change?

If the client is at commit n+1 and the server at n there will be problems